cdap-docs/_common/_source/index.rst

```diff
@@ -106,6 +106,12 @@ on the installation, monitoring and diagnosing fully distributed CDAP in a Hadoo
 - |faq|_ Answers to questions you might have about CDAP
 
 
+.. |search| replace:: **Search:**
+.. _search: search.html
+
+- |search|_ Search this documentation, using either *Quick Search* or *Google*
```

Comment:
Ah, I just saw the staged docs. I think it's kind of messy UI/UX, but probably fine for now.